### DISCUSSION TITLE
user ID error. Authentication error: Get accesstoken failed. Username or password error!

### DISCUSSION THREAD

**Discussion by cohen-sydney** [milo526/homebridge-tuya-web#500] [General]
TuyaWebPlatform] Authentication error: Get accesstoken failed. Username or password error!

I have the Fashin Smart light and use the Freshin Smart Life app.

is this plug in correct for this hardware and App??
I have entered the same User ID and PW as I use to access the App but get this error.
help is appreciated.

config redacted

"name": "TuyaWebPlatform",
    "options": {
        "username": "andrew@somedomain.net",
        "password": "[asaswrdnothere",
        "countryCode": "61",
        "platform": "smart_life"
    },
    "defaults": [
        {
            "id": "bf001863d12212db62a4b9",
            "device_type": "light",
            "light_characteristics": [
                "Brightness",
                "Color",
                "Color Temperature"
            ]
        }
    ],
    "scenes": false,
    "platform": "TuyaWebPlatform"

**Comment by cohen-sydney** [comment]
I have not setup an ID in
You will also need to create an account in the [Tuya IoT Platform](https://iot.tuya.com/).

do I need to do that???

**Comment by Nodgers** [comment]
I'm getting the same issue at my end :( out of interest, are you logged into the Smart Life app with your Apple ID? I'm wondering if that's why it's not accepting my username

**Reply by milo526** [reply]
Please check the [wiki page on authentication errors](https://github.com/milo526/homebridge-tuya-web/wiki/Authentication-Error)
